src/main/java/be/cytomine/client/models/SoftwareParameter.java

```diff
@@ -44,4 +46,42 @@ public SoftwareParameter(String name, String type, Long idSoftware, String defau
     public SoftwareParameter(String name, String type, Long idSoftware, String defaultValue, boolean required, int index, boolean setByServer){
         this(name, type, idSoftware, defaultValue, required, index, null, null, null, setByServer);
     }
+
```

Comment:
Idem as other. You don't implement the generic version of the client.

Reply:
done